src/Spryker/Zed/EventBehavior/Business/Model/TriggerManager.php

```diff
@@ -80,14 +90,20 @@ public function triggerRuntimeEvents()
 
         $processId = RequestIdentifier::getRequestId();
         try {
-            $events = $this->queryContainer->queryEntityChange($processId)->find()->getData();
-            static::$eventBehaviorTableExists = true;
-        } catch (PropelException | ConnectionException | EventBehaviorQueryNotExistsException $e) {
+            if(!$this->eventBehaviorTableExists()){
+                static::$eventBehaviorTableExists = false;
+
+                return;
+            }
+        } catch ( \Throwable $exception) {
```

Comment:
We shouldn't silently ignore exceptions, especially if we no longer expect any (We do the check for table/query existence beforehand).
Please leave only `ConnectionException` here.